packages/apollo-server-cache-redis-cluster/src/RedisClusterCache.ts

```diff
@@ -0,0 +1,56 @@
+import { KeyValueCache } from 'apollo-server-caching';
+import Redis, {
+  ClusterOptions,
+  ClusterNode,
+  Redis as RedisInstance,
+} from 'ioredis';
+import DataLoader from 'dataloader';
+
+export class RedisClusterCache implements KeyValueCache {
+  readonly client: any;
+  readonly defaultSetOptions = {
+    ttl: 300,
+  };
+
+  private loader: DataLoader<string, string>;
+
+  constructor(nodes: ClusterNode[], options?: ClusterOptions) {
+    this.client = new Redis.Cluster(nodes, options);
+
+    this.loader = new DataLoader(
+      (keys = []) =>
+        Promise.all(
+          keys.map(key => this.client.get(key).catch((error: Error) => error)),
```

Comment:
Sorry, according to the [HTTPCache code](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-datasource-rest/src/HTTPCache.ts#L22-L23), it expects a value or undefined, so your first code `.catch(() => null)` was correct.
Or catch the exception in the get method like this:

```typescript
  async get(key: string): Promise<string | undefined> {
    try {
      const reply = await this.loader.load(key);
      ...
    } catch {
      return;
    }
```

Reply:
Yeah. I prefer the `() => null` way. Done!